pkgs/by-name/ro/root/package.nix

```diff
@@ -135,6 +135,8 @@ stdenv.mkDerivation rec {
       hash = "sha256-D7LZWJnGF9DtKcM8EF3KILU81cqTcZolW+HMe3fmXTw=";
       revert = true;
     })
+    # Will also be integrated to ROOT 6.38.00
+    ./Build-rootcint-and-genreflex-as-separate-targets.patch
```

Comment:
Please use `fetchpatch2`, it should be fine to refer to a commit within unmerged upstream PR by sha.